src/os/CMakeLists.txt

```diff
@@ -115,16 +120,22 @@ endif()
 
 if(WITH_SPDK)
   include_directories("${CMAKE_SOURCE_DIR}/src/spdk/include")
+  add_custom_target(build_dpdk
```

Comment:
I prefer we uses submodule by default. just like boost submodule reason, distro version isn't reliable.